core/src/main/scala/cats/data/XorT.scala

```diff
@@ -219,6 +219,21 @@ trait XorTFunctions {
 
   final def pure[F[_], A, B](b: B)(implicit F: Applicative[F]): XorT[F, A, B] = right(F.pure(b))
 
+  /**
+   * Alias for [[XorT.right]]
+   * {{{
+   * scala> import cats.data.XorT
+   * scala> import cats.instances.option._
```

Comment:
Could you please use `import cats.implicits._` here? See #1026 for justification.